README.md

```diff
@@ -42,37 +49,63 @@ Ensure you have the following installed:
    environments.
 3. **[Docker](https://docs.docker.com/engine/install/)**
 4. **Operation System**: Ubuntu/MacOS
+5. **[Node](https://nodejs.org/en)** and **[`nvm` (Node Version Manager)](https://github.com/nvm-sh/nvm)** for front-end tooling.
```

Comment:
Can we also add Postgres? I had an issue with some dependencies and commands by `psycopg2`. Resolved by installing postgres `brew install postgresql`, or have I missed a trick?

Reply:
This falls under local running. Let's tackle that separately